api/v1beta1/auth_config_types.go

```diff
@@ -774,6 +774,7 @@ func (s *AuthConfigStatus) Ready() bool {
 // AuthConfig is the schema for Authorino's AuthConfig API
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
```

Comment:
considered adding `kubebuilder:deprecatedversion` ??

Reply:
Should not be done in this step. check my [comment](https://github.com/Kuadrant/authorino/pull/417#discussion_r1328779064) above.